core/types.go

```diff
@@ -27,6 +32,7 @@ type Authorization struct {
 	ID          string
 	URL         string
 	ExpiresDate time.Time
+	Order       *Order
```

Comment:
It's used by the WFE's `updateChallenge` function where a POST to an Authz checks the Order the authz is associated with to determine if it is expired. If it is then POSTing the Authz isn't allowed.

Should it be a `[]*Order` field? Right now pebble creates authorizations uniquely per order. You can't have an authz shared across orders (at least right now).